pkg/util/winutil/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/DataDog/datadog-agent/pkg/util/winutil
```

Comment:
We should probably update or address this deprecation notice: https://pkg.go.dev/github.com/DataDog/datadog-agent/pkg/util/winutil